client/src/main.rs

```diff
@@ -11,18 +23,27 @@ use accord::{ENC_TOK_LEN, SECRET_LEN};
 
 use std::net::SocketAddr;
 
-use tokio::sync::oneshot;
+use tokio::sync::{mpsc, oneshot};
 
 use rand::{rngs::OsRng, Rng, SeedableRng};
 use rand_chacha::ChaCha20Rng;
 
 use rsa::PaddingScheme;
 use rsa::PublicKey;
 
+use crate::console::ConsoleMessage;
+
+#[cfg(target_os = "unix")]
+const OS_EOL: &[u8] = b"\n";
+#[cfg(target_os = "windows")]
+const OS_EOL: &[u8] = b"\r\n";
```

Comment:
It's better to do `#[cfg(unix)]` and `#[cfg(windows)]`.
There is no `target_os="unix"`. There is `"linux"`, but you probably meant `target_family` instead. `#[cfg(unix)]` is just a shorthand version of that.
See [`target_os`, `target_family` and `unix and windows`](https://doc.rust-lang.org/reference/conditional-compilation.html#target_os) for more information.